broker/cmd/dejaqcli-broker/main.go

```diff
@@ -0,0 +1,161 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"net"
+	"os"
+	"os/signal"
+	"time"
+
+	"github.com/ilyakaznacheev/cleanenv"
+
+	"github.com/dejaq/prototype/broker/pkg/carrier"
+	"github.com/dejaq/prototype/broker/pkg/overseer"
+	"github.com/dejaq/prototype/broker/pkg/storage/cockroach"
+	"github.com/dejaq/prototype/broker/pkg/storage/inmemory"
+	"github.com/dejaq/prototype/broker/pkg/storage/redis"
+	storageTimeline "github.com/dejaq/prototype/broker/pkg/storage/timeline"
+	"github.com/dejaq/prototype/grpc/DejaQ"
+	flatbuffers "github.com/google/flatbuffers/go"
+	_ "github.com/lib/pq"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+)
+
+type Config struct {
+	Host         string `env:"HOST" env-default:"127.0.0.1:9000"`
+	OverseerSeed string `env:"OVERSEER" env-default:"localhost:9000"`
+	StorageType  string `env:"STORAGE_TYPE" env-default:"memory"`
+	StorageHost  string `env:"STORAGE_HOST"`
+
+	TopicCount        int `env:"TOPIC_COUNT"`
```

Comment:
I suggest to remove these 3 settings and the concurrent Streams limitations, or set a specific config for it

Reply:
Remove those settings, add "MaxConnectionsLimit"